### DISCUSSION TITLE
prompt font color changes

### DISCUSSION THREAD

**Discussion by brndngrhm** [posit-dev/positron#4443] [Q&A]
before typing, the prompt is a dark gray color:

![image](https://github.com/user-attachments/assets/ee3a22bd-2f24-4d31-8971-a664c183bcb5)

The when i start typing the font color changes: 

![image](https://github.com/user-attachments/assets/158b0da4-5f8b-4cc7-a513-cd6c32e779ef)

then after the code finishes it goes back to the darker gray color: 

![image](https://github.com/user-attachments/assets/6948bfcb-aa9a-4c40-bd58-23c51bfa93ca)

The dark gray font color on the dark backround sometimes makes it hard to tell when the code has completed at quick glance. Tried searching for a setting to adjust this but I couldn't find one, is there a way to change this?

**Comment by juliasilge** [comment]
I believe this is the same as #3970. What do you think? Also, what OS are you using? We were only able to reproduce this on Linux internally.

**Comment by brndngrhm** [comment]
yeah that looks like its the same thing. I'm on windows 10, default theme, on version `Positron-2024.08.0-48-UserSetup`

**Comment by softwarenerd** [comment]
@brndngrhm We found and fixed this issue for Linux builds of Positron. It turned out that the `CodeEditorWidget` we use in the Console was intentionally dimming the prompt on Linux. A PR is in the works for this fix.

You report that this was also happening to you on Windows 10. We have been unable to reproduce this issue on Windows 11, so we're working on getting a Windows 10 test environment running.

Are you certain that the screen shots you provided were from a Windows system? The font in them appears to be Droid Sans Mono, so I'm wondering about that. 

Thanks!

**Reply by brndngrhm** [reply]
Yes i am on my work HP laptop running windows 10 enterprise with the following Positron version (i also have fira code set for my terminal font in case that clarifies why it looked different):

```
Positron Version: 2024.09.0 (user setup) build 
Code - OSS Version: 1.92.0
Commit: f37f4f5044a2a619e73d5db61a31e37fbd3faf18
Date: 2024-09-03T02:39:11.525Z
Electron: 30.1.2
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.19045
```
